cmd/termui/view/termuic/termuiRenders/widgetsRender.go

```diff
@@ -162,6 +165,8 @@ func (wr *WidgetsRender) prepareInstanceInfo() {
 	countAcceptedBlocks := wr.presenter.GetCountAcceptedBlocks()
 	rows[4] = []string{fmt.Sprintf("Blocks proposed: %d | Blocks accepted:  %d", countLeader, countAcceptedBlocks)}
 
+	rows[5] = []string{computeRedundancyStr(wr.presenter.GetRedundancyLevel(), wr.presenter.GetRedundancyIsMainActive())}
+
 	// TODO: repair the rewards estimation or replace these 2 rows with rating details
```

Comment:
I do not think this todo is still relevant and the code can be removed. Rewards are visible on the explorer

Reply:
right. removed